content/en/docs/ambient/upgrade/helm/index.md

```diff
@@ -214,7 +214,7 @@ $ helm upgrade istio-ingress istio/gateway -n istio-ingress
 If you have upgraded all data plane components to use the new revision of the Istio control plane, and are satisfied that you do not need to roll back, you can remove the previous revision of the control plane by running:
 
 {{< text syntax=bash snip_id=none >}}
```

Comment:
Really these should use doctests, which would prevent a lot of this stuff being out of sync.